app/controllers/admin/routes.py

```diff
@@ -76,6 +76,7 @@ def createEvent(templateid, programid=None):
     eventData = template.templateData
     if request.method == "POST":
         attachmentFiles = request.files.getlist("attachmentObject")
+        fileDoesNotExist = attachmentFiles[0].content_type == "application/octet-stream"
```

Comment:
Could we just make attachmentFiles = None here if there are no files? We wouldn't need a conditional later